mw-webapp/src/dataAccessLogic/getDayReports.ts

```diff
@@ -41,10 +43,21 @@ export const getDayReports = async (): Promise<DayReport[]> => {
       return problemForCurrentPeriod;
     });
 
+  const mentorComments = firstReport.mentorComments.
+    map((mentorCommentUuid) => {
+      const mentorComment = mentorCommentsPreview
```

Comment:
one line. 
Check rules (eslint, prettier?)

Reply:
Dont understand what you mean, just copied same style for other variable